packages/run-protocol/src/econ-behaviors.js

```diff
@@ -442,3 +443,158 @@ export const startRewardDistributor = async ({
+    consume: {
+      zoe,
+      // ISSUE: is there some reason Zoe shouldn't await this???
+      feeMintAccess: feeMintAccessP,
+      runStakeBundle,
+      lienBridge,
+      client,
+      chainTimerService,
+      economicCommitteeCreatorFacet,
+    },
+    // @ts-ignore TODO: add to BootstrapPowers
+    produce: { runStakeCreatorFacet, runStakeGovernorCreatorFacet },
+    installation: {
+      consume: { contractGovernor },
+      produce: { runStake: runStakeinstallR },
+    },
+    instance: {
+      consume: { economicCommittee },
+      produce: { runStake: runStakeinstanceR },
+    },
+    brand: {
+      consume: { BLD: bldBrandP, RUN: runBrandP },
+      produce: { Attestation: attestationBrandR },
+    },
+    issuer: {
+      consume: { BLD: bldIssuer },
+      produce: { Attestation: attestationIssuerR },
+    },
+  },
+  config = {
+    mintingRatio: [1n, 4n],
+    interestRateBP: 250n,
+    loanFeeBP: 200n,
+    chargingPeriod: SECONDS_PER_HOUR,
+    recordingPeriod: SECONDS_PER_DAY,
+  },
+) => {
+  const bundle = await runStakeBundle;
+  const [feeMintAccess, bldBrand, runBrand, governor, installation, timer] =
+    await Promise.all([
+      feeMintAccessP,
+      bldBrandP,
+      runBrandP,
+      contractGovernor,
+      E(zoe).install(bundle),
+      chainTimerService,
+    ]);
+
+  const installations = {
+    governor,
+    getRUN: installation,
+  };
+
+  const poserInvitationP = E(
+    economicCommitteeCreatorFacet,
+  ).getPoserInvitation();
+  const [initialPoserInvitation, electorateInvitationAmount] =
+    await Promise.all([
+      poserInvitationP,
+      E(E(zoe).getInvitationIssuer()).getAmountOf(poserInvitationP),
+    ]);
+
+  // t.log({ electorateCreatorFacet, electorateInstance });
+  const runStakeTerms = makeRunStakeTerms(
+    {
+      timerService: timer,
+      chargingPeriod: config.chargingPeriod,
+      recordingPeriod: config.recordingPeriod,
+    },
+    {
+      mintingRatio: makeRatio(
+        config.mintingRatio[0],
+        runBrand,
+        config.mintingRatio[1],
+        bldBrand,
+      ),
+      interestRate: makeRatio(config.interestRateBP, runBrand, BASIS_POINTS),
+      loanFee: makeRatio(config.loanFeeBP, runBrand, BASIS_POINTS),
+      electorateInvitationAmount,
+    },
+  );
+
+  /** @type {{ publicFacet: GovernorPublic, creatorFacet: GovernedContractFacetAccess<any>}} */
+  const governorFacets = await E(zoe).startInstance(
+    installations.governor,
+    {},
+    {
+      timer,
+      economicCommittee,
+      governedContractInstallation: installations.getRUN,
+      governed: harden({
+        terms: runStakeTerms,
+        issuerKeywordRecord: { Stake: bldIssuer },
+        privateArgs: { feeMintAccess, initialPoserInvitation, lienBridge },
+      }),
+    },
+    harden({ economicCommitteeCreatorFacet }),
+  );
+
+  const governedInstance = await E(governorFacets.creatorFacet).getInstance();
+  const creatorFacet = E(governorFacets.creatorFacet).getCreatorFacet();
```

Comment:
creatorFacet.getCreatorFacet() is quite confusing. @Chris-Hibbert maybe that's another thing for https://github.com/Agoric/agoric-sdk/issues/4818